### PR TITLE
【infrt】add TrtOpConverterPass

### PR DESCRIPTION
**Pull request by shangzhizhou**
### PR types
New features

### PR changes
Others

### Describe
add TrtOpConverterPass to InfRT：
Input ir：
```
module  {
  func @main() -> tensor<?xf32> {
    %0 = pd.feed() {name = "input0"} : tensor<?xf32>
    %1 = pd.feed() {name = "input1"} : tensor<?xf32>
    %2 = pd.feed() {name = "input2"} : tensor<?xf32>
    %3 = pd.feed() {name = "input3"} : tensor<?xf32>
    %4 = pd.feed() {name = "input4"} : tensor<?xf32>
    %5 = pd.feed() {name = "input5"} : tensor<?xf32>
    %6 = "pd.graph"(%1, %0, %2, %4, %3, %5) ( {
      %7 = "pd.elementwise_add"(%1, %0) {axis = 1 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %8 = "pd.relu6"(%7) : (tensor<?xf32>) -> tensor<?xf32>
      %9 = "pd.matmul"(%8, %2) {transpose_x = false, transpose_y = false} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %10 = "pd.elementwise_add"(%9, %4) {axis = 1 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %11 = "pd.relu"(%10) : (tensor<?xf32>) -> tensor<?xf32>
      %12 = "pd.matmul"(%11, %3) {transpose_x = true, transpose_y = false} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %13 = "pd.elementwise_add"(%12, %5) {axis = 1 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %14 = "pd.relu"(%13) : (tensor<?xf32>) -> tensor<?xf32>
      "pd.return"(%14) : (tensor<?xf32>) -> ()
    }) : (tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
    "pd.fetch"(%6) {name = "output"} : (tensor<?xf32>) -> ()
  }
}
```
Output ir:
```
module  {
  func @main() -> tensor<?xf32> {
    %0 = pd.feed() {name = "input0"} : tensor<?xf32>
    %1 = pd.feed() {name = "input1"} : tensor<?xf32>
    %2 = pd.feed() {name = "input2"} : tensor<?xf32>
    %3 = pd.feed() {name = "input3"} : tensor<?xf32>
    %4 = pd.feed() {name = "input4"} : tensor<?xf32>
    %5 = pd.feed() {name = "input5"} : tensor<?xf32>
    %6 = "pd.graph"(%1, %0, %2, %4, %3, %5) ( {
      %7 = "trt.ElementWise"(%1, %0) {elementwise_operation = 0 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %8 = "trt.Activation"(%7) {activation_type = 8 : si32, alpha = 0.000000e+00 : f32, beta = 6.000000e+00 : f32} : (tensor<?xf32>) -> tensor<?xf32>
      %9 = "trt.MatrixMultiply"(%8, %2) {transpose1 = false, transpose2 = false} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %10 = "trt.ElementWise"(%9, %4) {elementwise_operation = 0 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %11 = "trt.Activation"(%10) {activation_type = 0 : si32, alpha = 0.000000e+00 : f32, beta = 0.000000e+00 : f32} : (tensor<?xf32>) -> tensor<?xf32>
      %12 = "trt.MatrixMultiply"(%11, %3) {transpose1 = false, transpose2 = false} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %13 = "trt.ElementWise"(%12, %5) {elementwise_operation = 0 : si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
      %14 = "trt.Activation"(%13) {activation_type = 0 : si32, alpha = 0.000000e+00 : f32, beta = 0.000000e+00 : f32} : (tensor<?xf32>) -> tensor<?xf32>
      "pd.return"(%14) : (tensor<?xf32>) -> ()
    }) : (tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
    "pd.fetch"(%6) {name = "output"} : (tensor<?xf32>) -> ()
  }
}
```

### CODE REVIEW

File: paddle/infrt/dialect/tensorrt/trt_op_converter_pass.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ *   %d, %f = "pd.graph"(%a) {
+ *     %m = "trt.Convolution"(%a)...
+ *     %n = "trt.Convolution"(%m)...
+ *     %s = "trt.Convolution"(%a)...
+ *     "pd.return" %n, %s
+ *   } ...
+ *   "pd.fetch" %d, %f
+ * }
+ */
+struct trtOpConverterPass
```
Comment (Superjomn) [inline paddle/infrt/dialect/tensorrt/trt_op_converter_pass.h]: `TRTOpConvertPass` for that this class is defined from scratch, not generated from TableGen.

Comment (shangzhizhou) [reply paddle/infrt/dialect/tensorrt/trt_op_converter_pass.h]: done.

**Review by winter-wang — DISMISSED** [review]
LGTM

**Review by winter-wang — APPROVED** [review]
LGTM

**Pull request merged**